sass/stylesheet.scss

```diff
@@ -1,3 +1,60 @@
-body{
-  background-color: red;
-}
\ No newline at end of file
+@import "component";
+
+
+.card{
+  border:none;
+}
+
+
+.profile-img{
+  border-radius: 50%;
```

Comment:
There is a set of Bootstrap classes for border radius if you want to use that instead